src/components/common/ui/tooltip-mui/Tooltip.styles.tsx

```diff
@@ -0,0 +1,17 @@
+import { styled } from '@mui/material/styles';
+import Tooltip, { tooltipClasses, TooltipProps } from '@mui/material/Tooltip';
+
+export const StyledTooltip = styled(({ className, ...props }: TooltipProps) => (
+  <Tooltip {...props} classes={{ popper: className }} />
+))(({ theme }) => ({
+  [`& .${tooltipClasses.tooltip}`]: {
+    p: '4px 12px',
+    textAlign: 'center',
+    borderRadius: '4px',
+    backgroundColor: '#FFE0E0',
+    color: '#151515',
+  },
+  [`& .${tooltipClasses.arrow}`]: {
+    color: '#FFE0E0',
```

Comment:
Use colors from theme. If you use sx you can access them just by string value, like: 'primary.300', 'grey.500', etc.